### DISCUSSION TITLE
What is necessary to accept followers?

### DISCUSSION THREAD

**Discussion by riedel** [snarfed/bridgy-fed#1132] [Q&A] [answered]
I created an account via an rss feed (@teco.edu@teco.edu) and tried to follow it.

Follow button is there but the process seems to get stuck:
![grafik](https://github.com/snarfed/bridgy-fed/assets/566485/a7ff46bb-642f-4cc5-b0a5-ee0cae7da361)

Is there anything one needs to do to accept followers?

**Comment by qazmlp** [comment]
The website must implement microformats2 to some extent to be used with Bridgy Fed.
Start reading at https://fed.brid.gy/docs#web-profile for more information about that.

I'm note sure if it's for that reason specifically, but it's possible the follow is blocked because no profile info could be set up.
(@snarfed It might be better to error out entirely if neither the profile nor posts have microformats during initial setup.)

**Reply by snarfed** [reply]
Microformats are only required for webmentions. BF can happily read post content from RSS/Atom and profile info from OGP or other HTML metadata.

**Comment by snarfed** [comment, marked as answer]
Hmm! Sorry for the trouble. Not sure what's going on here, but I can look.

**Reply by riedel** [reply]
Would be great,thanks. Unfortunately, I do not well understand the protocol. I was at first having the suspicion that I block some (POST, etc) call to the .well-known URLs, however, I could not see any incoming traffic when pressing the follow button.

**Reply by snarfed** [reply]
Aha. Looking at the logs, it looks like the Mastodon account you're using is getting caught by Bridgy Fed's spam filters because it's not complete enough: it has no profile picture, its display name is the same as its username, and it's only two days old. https://fed.brid.gy/docs#fediverse-cant-find

**Reply by riedel** [reply]
OK, thanks so much for clarifying. So one needs to have a 2 weeks old account to follow another fed bridgy account? I'll wait 2 weeks. However, to me this seems to be not optimal setup if one wants to gain reach/followers. Maybe the spam filter should let the follow request through in such cases.

**Reply by riedel** [reply]
@snarfed , can you confirm this is still the reason (sorry for nagging, but we need are currently considering rebuilding our whole website and some fediverse feed is part of it andy bridgy-fed seemed like a really easy way to do it). 

Now after 2 weeks my fediverse rookie account (@riedel@sigmoid.social) still seems to be blocked by @teco.edu@teco.edu. Is the spam filter maybe stateful? 

To say it again: As we are doing education, the 14-day spam rule applied to following might have strong adversarial effects if we are eg. telling students: just get a fediverse account and follow us (telling them to first wait 14 days and get credibility for their account before they can follow you, might be a bit strange)

**Reply by snarfed** [reply]
Hmm! Try unfollowing it and then follow again? 

Also, if you want students to be able to follow your web site immediately, Bridgy Fed may not be the best approach. Maybe consider having them use an RSS feed reader or something similar instead?

**Reply by riedel** [reply]
Thanks for the prompt reply. Have you ever tried to tell an 20 year old what an RSS reader is ;) I think I'd rather give up for now then. But in general fed bridgy is a really cool concept.

**Reply by qazmlp** [reply]
@riedel For what it's worth, [RSS Parrot](https://rss-parrot.net/) exists too, if you just want push notifications for RSS items.

That's really the only thing it does, though. The implementation is so barebones that boosted posts may not appear on other instances in some cases, I assume because they can't be fetched.
